picard/album.py

```diff
@@ -163,7 +163,21 @@ def _release_request_finished(self, document, http, error):
         self.load_task = None
         parsed = False
         try:
-            if error:
+            if (error
+                and config.setting['track_ars']
+                and not self.gateway_timeout
+                and repr(http.attribute(QtNetwork.QNetworkRequest.HttpStatusCodeAttribute)) == "504"):
```

Comment:
It is so long since I wrote this I am not sure why I put the repr in rather than simply compare `http.attribute(QtNetwork.QNetworkRequest.HttpStatusCodeAttribute) == 504`.

I obviously thought it was required at the time - but I suspect it may have been because `http.attribute(QtNetwork.QNetworkRequest.HttpStatusCodeAttribute)` is of type QVariant.Int (see [here](http://pyqt.sourceforge.net/Docs/PyQt4/qnetworkrequest.html)) and I wanted to be sure that the code didn't cause an exception by making it a string comparison.